src/i18n/fi.json

```diff
@@ -129,6 +130,8 @@
     "validation-stringLengthCounter": " merkkiä jäljellä",
     "validation-requiredInContentLanguages": "Tämä kenttä vaaditaan kaikilla valituilla kielillä",
     "validation-offerIsFreeOrHasPrice": "Maksullinen tapahtuma vaatii hinnan kaikilla valituilla kielillä",
+    "validation-atLeastOneIsTrue": "Valitse vähintään yksi toistopäivä",
+    "validation-isMoreThanOne": "Toistovälin tulee olla vähintään 1 viikko",
```

Comment:
Which message do you mean? For me, these messages show up when:
1. Open the recurring modal
2. Empty "toistoväli"-input field
3. Make sure all checkboxes are unchecked
4. Click "Lisää kerrat"-button